pocs/hardware.py

```diff
@@ -1,6 +1,14 @@
 """Information about hardware supported by Panoptes."""
 
-ALL_NAMES = sorted(['camera', 'dome', 'mount', 'night', 'weather', 'theskyx'])
+ALL_NAMES = sorted([
+    'camera',
+    'dome',
+    'mount',
+    'night',
```

Comment:
Along with my suggestion that we add a `power_failure` simulator, we could also add `day` and `storm` simulators.

We probably need different variables here to hold different kinds of names: equipment names vs. condition names. `day` and `night`, or `has_ac` and `ac_failure` would represent conditions, while `camera`, `dome`, etc. represent equipment.

Reply:
I think we should discuss this more and figure out what we want. I agree there should be some logical distinction for the simulator types. For this PR I will leave the simple `power` simulator.

Reply:
Agreed on both fronts.